src/main/java/lotto/view/InputView.java

```diff
@@ -0,0 +1,22 @@
+package lotto.view;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+public class InputView {
+	private static final Scanner scanner = new Scanner(System.in);
+	public static int inputBuyMoneyAndReturnInputValue() {
+		System.out.println("구입금액을 입력해 주세요.");
+		return scanner.nextInt();
+	}
+
+	public static List<Integer> inputWinLottoNumbersAndReturnInputValue() {
+		System.out.println("지난 주 당첨 번호를 입력해 주세요.");
+		scanner.skip("[\r\n]+");
+		String input = scanner.nextLine();
+		String[] inputArray = input.split(",");
+		return Arrays.stream(inputArray).map(i -> i.trim()).map(Integer::parseInt).collect(Collectors.toList());
```

Comment:
```suggestion
		return Arrays.stream(input.split(DELIMITER))
				.map(String::trim)
				.map(Integer::parseInt)
				.collect(Collectors.toList());
```
한줄로 쓰면 협업할 때 코드 읽기 힘드니 적절히 엔터를 넣어주세요